squire_lib/src/accounts.rs

```diff
@@ -0,0 +1,20 @@
+use uuid::Uuid;
+use settings;
+
+struct squire_account {
+    display_name: String,
+    user_name: String,
+    user_id: Uuid,
```

Comment:
In general, I prefer to no use plain `Uuid`s. I generally prefer to wrap them in a struct for type safety. This allows us to specific what kind of id we expect in function signatures and sure. There is a file that defines the identifiers used in `squire_lib`, `identifiers.rs`. I would add Id types for both account types and make their id fields use those types.